### ISSUE TITLE
Support ignoring Unreal content

### ISSUE THREAD

**Issue by gryphonmyers** [ProjectBorealis/UE4GitPlugin#79]
We ignore our Developers folders (currently located at Content/Developers/{developername}/) because we have no need to persist the things we do in them. This means we have an entry in our .gitignore, which successfully keeps anything within from showing up in git command line. However, the files still have all the built in source control handling within Unreal, even though none of it works. For example, we can still "Check Out" these ignored files from within the Unreal UI, but then we get lines like the following in the output log:

`LogSourceControl: Error: Command 'Revert' Failed!`
`LogSourceControl: Error: Command 'Delete' Failed!`

I also attempted adding exclusions to .gitattributes making them non-lockable, not tracked by lfs, but that doesn't appear to have changed anything with regard to Unreal's attempts at managing these files as source controlled.

I'm wondering if there is some piece of configuration that I am lacking here to get these files to be fully ignored by Unreal / this plugin, or if this is a use case that it is not prepared for?

**Comment by mastercoms** [comment]
Yeah we don't expect .uassets or .umaps in content folders to be ignored. I will see what I can do, thanks for letting me know about this!

**Comment by MalikuMane** [comment]
I have ignored folders too and upon adding an asset i get "LogSourceControl: Error: Command 'MarkForAdd' Failed!". I would expect that there should be no attempt to MarkForAdd if the ignore pattern is matched.
